src/pages/Signup.jsx

```diff
@@ -0,0 +1,64 @@
+import { useState } from "react";
+import { Link } from "react-router-dom";
+import { useAuth } from "../contexts";
+
+function Signup() {
+  const [name, setName] = useState("");
+  const [email, setEmail] = useState("");
+  const [pass, setPass] = useState("");
+
+  const { userSignup } = useAuth();
+
+  return (
+    <div className="center-hv">
+      <div className="auth-form">
+        <form onSubmit={(e) => userSignup(e, email, pass, name)}>
+          <h2>Sign up</h2>
+          <div className="inp-unit">
+            <label for="email">Name</label>
+            <input
+              type="text"
+              name="email"
+              placeholder="johnsangram"
+              onChange={(e) => setName(e.target.value)}
+            />
+          </div>
+          <div className="inp-unit">
+            <label for="email">Email</label>
+            <input
+              type="email"
+              name="email"
+              placeholder="johnsangram@gkool.com"
+              onChange={(e) => setEmail(e.target.value)}
+            />
+          </div>
+          <div className="inp-unit">
+            <label for="email">Password</label>
+            <input
+              type="password"
+              name="email"
```

Comment:
type is password but name is email. Check this out

Reply:
I'll look into it and fix it.